j2lint/linter/rule.py

```diff
@@ -14,15 +13,22 @@ class Rule:
     """
     id = None
     description = None
-    check = None
-    checktext = None
     ignore = False
     warn = []
 
     def __repr__(self):
         return self.id + ": " + self.description
 
-    def is_valid_language(self, file):
+    @classmethod
+    def checktext(cls, file, text):
+        """ class method for checktext """
+
+    @classmethod
```

Comment:
Why make these class methods? they are overwritten as method in the various rule files? Please also document in the docstring that you expect a Rule to return a certain format when implementing this method.

Reply:
I have corrected the docstring. For this , I was getting a warning "checktext/check is not callable" that's why added these class methods.

Reply:
Added disable for this warning and removed class method.